Aquarius/Src/Aquarius/Events/KeyboardEvent.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "Event.h"
+
+#include <GLFW/glfw3.h>
+
+
+namespace Aquarius {
+
+    class KeyEvent : public Event
+    {
+    public:
+
+        int getCode()
```

Comment:
This method should be const, and we should be using Input::KeyCode enum for the keycode member

Generally, any method that won't modify the class should be marked  const